### PR TITLE
Convert from docbook 4.5 -> 5.0

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: guide/resources/jquery-2.1.1.min.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
...ribute(c)},attrHooks:{type:{set:function(a,b){if(!k.radioValue&&"radio"===b&&n.nodeName(a,"input")){var c=a.value;return a.setAttribute("type",b),c&&(a.value=c),b}}}}}),Zb={set:function(a,b,c){return b===!1?n.removeAttr(a,c):a.setAttribute(c,c),c}},n.each(n.expr.match.bool.source.match(/\w+/g),function(a,b){var c=$b[b]||n.find.attr;$b[b]=function(a,b,d){var e,f;return d||(f=$b[b],$b[b]=e,e=null!=c(a,b,d)?b.toLowerCase():null,$b[b]=f),e}});var _b=/^(?:input|select|textarea|button)$/i;n.fn.extend({prop:function(a,b){return J(this,n.prop,a,b,arguments.length>1)},removeProp:function(a){return this.each(function(){delete this[n.propFix[a]||a]})}}),n.extend({propFix:{"for":"htmlFor","class":"className"},prop:function(a,b,c){var d,e,f,g=a.nodeType;if(a&&3!==g&&8!==g&&2!==g)return f=1!==g||!n.isXMLDoc(a),f&&(b=n.propFix[b]||b,e=n.propHooks[b]),void 0!==c?e&&"set"in e&&void 0!==(d=e.set(a,c,b))?d:a[b]=c:e&&"get"in e&&null!==(d=e.get(a,b))?d:a[b]},propHooks:{tabIndex:{get:function(a){return a.hasAttribute("tabindex")||_b.test(a.nodeName)||a.href?a.tabIndex:-1}}}}),k.optSelected||(n.propHooks.selected={get:function(a){var b=a.parentNode;return b&&b.parentNode&&b.parentNode.selectedIndex,null}
```
Comment (raimue) [inline guide/resources/jquery-2.1.1.min.js]: Not sure what was changed in jquery, but it seems unrelated.

File: guide/xml/guide.xml [hunk 2]
```diff
@@ -1,44 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- -*- coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4:tw=120
 -->
-<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
-"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V5.0//EN"
+"http://docbook.org/xml/5.0/dtd/docbook.dtd">
```
Comment (raimue) [inline guide/xml/guide.xml:5]: 5.0 or 5.1? Is this DTD correct?

**Review by raimue — COMMENTED** [review]
Also I cannot spot any changes to `guide/resources/images/trac-default.png`, artifact from automatic replace?

File: guide/xml/guide.xml (hunk 2 above)
Comment (grimreaper) [reply guide/xml/guide.xml:5]: 5.0. We don't (yet) have a port for 5.1

File: guide/resources/jquery-2.1.1.min.js (hunk 1 above)
Comment (grimreaper) [reply guide/resources/jquery-2.1.1.min.js]: fixed

**Comment by grimreaper** [comment]
hrm...  I do not yet understand the relationship between docbook and OASIS....
- 5.0 appears to have "have a specification" while 5.1 does not.
- docbook claims the current release is 5.1
- OASIS claims 5.0 is not yet released.

File: README [hunk 3]
```diff
@@ -1,7 +1,7 @@
 This is the MacPorts documentation directory.
 
 To generate it you have to checkout the entire "macports-guide" repository,
-and you need the "docbook-xsl", "docbook-xml-4.5" and "libxslt" ports.
+and you need the "docbook-xsl", "docbook-xml-5.1" and "libxslt" ports.
```
Comment (neverpanic) [inline README]: You're referencing the docbook-xml-5.1 port here.

Comment (grimreaper) [reply README]: fixed. I think y'all figured out my original intent to go to 5.1 before realizing that would be harder :)

**Review by neverpanic — APPROVED** [review]
